MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -45,7 +45,9 @@ Please see [Before Development](MODULE_EVALUATION_TEMPLATE#before-development) f
   * _This is not applicable to libraries_
 * Module descriptor MUST include interface requirements for all consumed APIs (3, 5)
   * _This is not applicable to libraries_
-* Third party dependencies use an Apache 2.0 compatible license (2)
+* Inclusion of third party dependencies complies with [ASF 3rd Party License Policy](https://apache.org/legal/resolved.html) (2)
+  * Uses README for [Category B Appropriately Labelled Condition](https://apache.org/legal/resolved.html#appropriately-labelled-condition)
+  * org.z3950.zing:cql-java is allowed, even if it is LGPL-2.1-only
```

Comment:
I see why you noted this exception, and it makes sense to me.  But is there any action we should take for 

> This criterium can be changed as soon as better third party requirements get proposed.

?